.travis.yml

```diff
@@ -8,13 +8,15 @@ env:
   matrix:
   - BUILDTOOL=autotools
   - BUILDTOOL=cmake
-  - BUILDTOOL=autotools_old_compilers
```

Comment:
Yeah. But the old-compilers "build" is the only one that's always really fast :P